swagger.yaml

```diff
@@ -1,42 +1,292 @@
 swagger: "2.0"
+
 info:
-  title: dp-image-api
-  description: "TODO"
-  version: 1.0.0
+  title: "Digital Publishing Image API"
+  description: "Provides read and write capabilities for image metadata"
+  version: "1.0.0"
   license:
     name: "Open Government Licence v3.0"
     url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+
 basePath: "/v1"
+
 schemes:
   - http
+
 tags:
-  - name: "hello"
+  - name: "image"
+
 paths:
-  /hello:
+
+  /images:
     get:
       tags:
-        - hello
-      summary: Example hello world endpoint
-      description: Returns an example payload.
+      - "image"
+      summary: "Get images filtered by collection id"
+      description: "Returns a list of images metadata filtered by a compulsory query parameter defining the collection ID"
+      parameters:
+      - $ref: '#/parameters/collection_id'
       produces:
-        - application/json
+      - "application/json"
+      security:
+      - FlorenceAPIKey: []
       responses:
         200:
-          description: OK
+          description: "A json object containing images which correspond to the provided collection ID"
+          schema:
+            $ref: '#/definitions/Images'
+        400:
+          description: "Invalid request, collection id was incorrect"
+        401:
+          description: "Unauthorised to view images metadata"
+        404:
```

Comment:
We probably won't need a `404` here as if no results match we can just return an empty list.

Reply:
I agree, if there is no image, we can say that we found the images, but it happens to be an empty list.